instrumentation/p6spy/src/main/java/brave/p6spy/TracingP6SpyOptions.java

```diff
@@ -1,32 +1,58 @@
 package brave.p6spy;
 
-import com.p6spy.engine.spy.P6SpyOptions;
+import com.p6spy.engine.logging.P6LogLoadableOptions;
+import com.p6spy.engine.logging.P6LogOptions;
+import com.p6spy.engine.spy.P6LoadableOptions;
 import com.p6spy.engine.spy.option.P6OptionsRepository;
+
+import javax.management.StandardMBean;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
-final class TracingP6SpyOptions extends P6SpyOptions {
+final class TracingP6SpyOptions extends StandardMBean
+    implements P6LoadableOptions, TracingP6SpyOptionsMBean {
+
   static final String REMOTE_SERVICE_NAME = "remoteServiceName";
   static final String INCLUDE_PARAMETER_VALUES = "includeParameterValues";
+  private static final Map<String, String> defaults = new LinkedHashMap();
 
-  final P6OptionsRepository optionsRepository;
+  private final P6OptionsRepository optionsRepository;
+  private final P6LogLoadableOptions logLoadableOptions;
+
+  static {
+    defaults.put(INCLUDE_PARAMETER_VALUES, Boolean.FALSE.toString());
+  }
 
-  TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
-    super(optionsRepository);
+  public TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
+    super(TracingP6SpyOptionsMBean.class, false);
+    logLoadableOptions = new P6LogOptions(optionsRepository);
     this.optionsRepository = optionsRepository;
   }
 
   @Override public void load(Map<String, String> options) {
-    super.load(options);
+    logLoadableOptions.load(options);
     optionsRepository.set(String.class, REMOTE_SERVICE_NAME, options.get(REMOTE_SERVICE_NAME));
-    optionsRepository.set(Boolean.class, INCLUDE_PARAMETER_VALUES, options.get(INCLUDE_PARAMETER_VALUES));
+    optionsRepository.set(Boolean.class, INCLUDE_PARAMETER_VALUES,
+        options.get(INCLUDE_PARAMETER_VALUES));
+  }
+
+  @Override
+  public Map<String, String> getDefaults() {
+    Map<String, String> allDefaults = new LinkedHashMap();
+    allDefaults.putAll(defaults);
+    allDefaults.putAll(logLoadableOptions.getDefaults());
+    return allDefaults;
+  }
+
+  public P6LogLoadableOptions getLogOptions() {
```

Comment:
PS If you want to add an interface, just make the interface itself package private. Then, you can add `@Override` tag for this and below. As long as types are package private, we're good. 

f we are broken on JMX and don't know it because there is no test, a good fix would be to add a test vs us guessing if it is broke or not. I suspect there's code out there somewhere that has a unit test for JMX registration even if we don't have any in this repo. I'd consider this part optional for this PR even if we are broken as ensuring JMX works can be separate